app/models/miq_provision_virt_workflow.rb

```diff
@@ -1000,6 +1015,10 @@ def setup_parameters_for_visibility_service(options)
     }
   end
 
+  def create_hash_struct_from_source(source, options)
+    create_hash_struct_from_vm_or_template(source, options)
```

Comment:
The follow up pull request fills in the rest of the logic. I guess I could turn it into an alias for the sake of this pull request but it's not going to stay that way.